src/tests/unit/tests/electron/common/unified-feature-flags.test.ts

```diff
@@ -82,4 +83,18 @@ describe('FeatureFlagsTest', () => {
             expect(detail.displayableDescription.length > 0).toBeTruthy();
         });
     });
+
+    test('web and electron feature flags with matching names should have the same purpose', () => {
+        const unifiedDetails = getAllFeatureFlagDetailsUnified();
+        const webDetails = getAllFeatureFlagDetails();
+
+        unifiedDetails.forEach(unifiedDetail => {
+            const matchingWebDetail = webDetails.find(
+                webDetail => webDetail.id === unifiedDetail.id,
+            );
+            if (matchingWebDetail !== undefined) {
```

Comment:
Any reason we can't be more assertive here? Like comparing the values of the objects entirely?

Reply:
Added an assert on displayableDescription as well, but I don't think we would need to require the other values in the object (defaultValue, isPreviewFeature, and forceDefault) to be the same between web and electron.